src/repr/scalar/mod.rs

```diff
@@ -844,6 +844,15 @@ impl Interval {
         s + ns
     }
 
+    /// Computes the nanoseconds of the interval.
```

Comment:
Ah, this was an intentional omission on my end since all of the other values returned `<this time unit> + <all smaller time units..>`, and I was imagining that was `part`. Probably overthinking it, so glad to make it consistent.